### DISCUSSION TITLE
Help with playlists

### DISCUSSION THREAD

**Discussion by ejordan9** [ubuntu-flutter-community/musicpod#738] [Q&A]
Hi guys, Can someone tell me how the playlist works? When I add a link, for example from Spotify the playlist does not work, what am I doing wrong?

https://github.com/ubuntu-flutter-community/musicpod/assets/125694310/7329d3bf-2b32-4abd-bb7a-131f9ad79adb

 BTW is there any way to edit the name of the playlist? so that it is not (ttps://open.spotify....)
 
![image](https://github.com/ubuntu-flutter-community/musicpod/assets/125694310/2b9fa15f-c092-4e4f-ba61-51730cb21d9f)

Thanks

**Comment by Feichtmeier** [comment]
Hi, this app has no connection to Spotify
Playlists are mainly for local audio files and there are a bunch of ways to fill it. From a queue, from Audio tracks or manually.

**Reply by ejordan9** [reply]
Sure, it's obvious, I had read in a blog that you could add the url's, sorry.

**Reply by Feichtmeier** [reply]
yes there is a manual option for podcast feeds or radio streams
but the app is mainly built around radiobrowser api to search your stations by name/tag/country/etc and creat your collection out of those searches or by podcasts from itunes/podcastindex and also for local audio :)
thanks for using the app and best regards (feel free to reach out here on the discussions any time)

**Comment by claudioabreu07** [comment]
Hello everyone, musicapop has stopped working on my system for a month now. Can you tell me if it is undergoing an update or has been discontinued?

**Reply by Feichtmeier** [reply]
hi
musicapop?
the app works fine here on macos and linux and windows
what version are you using?
no it is not discontinued at all, just real life needing some more time than usual :PO

**Reply by claudioabreu07** [reply]
I am an elementary os user.
I loved this app.

**Reply by Feichtmeier** [reply]
what version of MusicPod are you using? The flatpak?

**Comment by claudioabreu07** [comment]
I'm on the latest version of Elementary Os, and the app version is 0.3.7

**Reply by Feichtmeier** [reply]
> I'm on the latest version of Elementary Os, and the app version is 0.3.7

Okay that's the flatpak 
I'm sorry 😔 
The snap works flawlessly
If you can switch to the snap

**Reply by claudioabreu07** [reply]
Now everything is fine, using snap installation mode.
Thanks!
